bigquery/tables.rb

```diff
@@ -94,10 +90,9 @@ def import_table_data project_id:, dataset_id:, table_id:, row_data:
   # table_id   = "ID of the table to import data into"
   # row_data   = [{ column1: value, column2: value }, ...]
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.